frontend/src/metabase/css/dashboard.module.css

```diff
@@ -104,14 +70,6 @@
   transition: border 0.3s, background-color 0.3s;
 }
 
-:global(.Dash--editing .Card-title:first-of-type) {
```

Comment:
I found both `.Dash--editing` and `.Card-title` are still being used before this PR.

https://github.com/metabase/metabase/blob/afc28b95811276d042d5dc1a1eff05faafd9393e/frontend/src/metabase/dashboard/components/DashboardGrid.tsx#L589

https://github.com/metabase/metabase/blob/afc28b95811276d042d5dc1a1eff05faafd9393e/frontend/src/metabase/visualizations/components/LegendHeader.jsx#L61

https://github.com/metabase/metabase/blob/afc28b95811276d042d5dc1a1eff05faafd9393e/frontend/src/metabase/visualizations/visualizations/Scalar/Scalar.jsx#L221

Can you make sure the `margin-top: 0.5rem` is still applied? I haven't tried to test if this rule is even possible, but it would be nice to double check.